coopr-docs/docs/source/guide/quickstart/index.rst

```diff
@@ -408,15 +428,18 @@ Once creation is complete, the cluster is ready for use.
 
 For more information on your cluster, click on the name 'hadoop-quickstart' on the
 Clusters' home screen. On this cluster description screen, nodes are grouped together by the set
-of services that are available on them. To see node details, click on the white triangles next to each
-service set to expand the list. The expanded list shows a list of attributes for each node.
+of services that are available on them. To see node details, click on the descriptions
+next to the number of nodes ('2 noes with 5 services each') to expand the list. The
+expanded list shows a list of attributes for each node.
 
-.. figure:: /_images/quickstart/cluster_details.png
+.. figure:: /_images/quickstart/qs_cluster_details.png
     :align: center
     :width: 800px
     :alt: Cluster description and details
     :figclass: align-center
 
-In this example, there is 1 master node that contains the 'hbase-master', 'hadoop-hdfs-namenode', 'zookeeper-server', and
-'hadoop-yarn-resourcemenager' services. There are also 4 slave nodes that contain the 'hbase-regionserver', 'hadoop-yarn-nodemanager',
-and 'hadoop-hdfs-datanode' services.
+In this example, there is 1 master node that contains the 'base',
+'continuuity-sensu-monitoring', 'hadoop-hdfs-namenode', 'hadoop-yarn-resourcemanager',
```

Comment:
continuuity-sensu-monitoring is one of our internal services.  it's probably ok for now since this is just an illustrative example, but it would be good to remove in the future